lisp/mastodon-tl.el

```diff
@@ -217,14 +217,17 @@ Return value from boosted content if available."
 (defun mastodon-tl--more ()
   "Append older toots to timeline."
   (interactive)
-  (let* ((tl (mastodon-tl--timeline-name))
+  (let* ((point-before-update (point))
+	 (tl (mastodon-tl--timeline-name))
```

Comment:
Can you fix the spacing on this line? I know it's a little thing but might as well clean it up.